ipam/ipv6Ipam.go

```diff
@@ -0,0 +1,263 @@
+package ipam
+
+import (
+	"context"
+	"errors"
+	"net"
+	"os"
+	"regexp"
+	"runtime"
+
+	"github.com/Azure/azure-container-networking/common"
+	"github.com/Azure/azure-container-networking/log"
+	"github.com/Masterminds/semver"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/version"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+const (
+	defaultWindowsKubePath           = `c:\k\`
+	defaultWindowsKubeConfigFilePath = defaultWindowsKubePath + `config`
+	defaultLinuxKubeConfigFilePath   = "/var/lib/kubelet/kubeconfig"
+	defaultSubnetMaskSizeLimit       = "/120"
```

Comment:
can you add a comment what this used for ?

Reply:
added comment in latest revision